plugins/QnA/models/class.qnamodel.php

```diff
@@ -0,0 +1,126 @@
+<?php
+/**
+ * Class QnaModel
+ */
+class QnaModel extends Gdn_Model {
+    /**
+     * Update Q&A counts through the dba/counts endpoint.
+     *
+     * @param string $column
+     * @return array $results Formatted to match what "dba.js" expects
+     */
+    public function counts($column) {
+        $result = ['Complete' => true];
+
+        switch($column) {
+            // Discussion table, QnA column will be updated.
+            case 'QnA':
+                $request = Gdn::request()->get();
+                $result = $this->recalculateDiscussionQnABatches($request['NumberOfBatchesDone'] ?? 0, $request['LatestID'] ?? 0);
+                break;
+        }
```

Comment:
No default needed?  Throw an Exception?

Reply:
I added the switch because of how other counts work around Vanilla. It's also easier to add other counts to QnA if we need to in the future.